File: 11-employee-directory-angular-ashwith-p/Employee_Directory/src/models/Role.ts
export class Role
{
    Id:number

    Name:string        

    Department:number;

    Description?:string;

    location:number[];        

    constructor(args:any)
    {
        this.Id=args.id
        this.Name=args.name;
        this.Department=args.department;
        this.Description=args?.description;
        this.location=args.location;
    }
}

File: 9-employee-directory-ef-core-ashwith-p/Database_First/Bussiness_Logic/DTO/Role.cs
namespace Domain.DTO
{
    public class Role
    {
        public string Name { get; set; } = string.Empty;        

        public string Department { get; set; } = string.Empty;

        public string? Description { get; set; }

        public List<string> Location { get; set; } = [];        
    }
}
